From 8373d5f5a1c9604a1a87cdc04668000720fe3155 Mon Sep 17 00:00:00 2001
From: Nikolai Kostrigin <nickel@altlinux.org>
Date: Fri, 13 May 2022 17:13:00 +0300
Subject: [PATCH] hwdb: fix accelerometer mount matrix for Aquarius NS483

Signed-off-by: Nikolai Kostrigin <nickel@altlinux.org>
(cherry picked from commit e4bb1a07ef834f4e4eabe1fa21e838586e7dd2a3)

Related: #2087778
---
 hwdb.d/60-sensor.hwdb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 3fb308f3f1..104fe05187 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -106,6 +106,14 @@ sensor:modalias:acpi:BOSC0200*:dmi:*svn*Acer*:*pn*Spin*SP111-33:*
 sensor:modalias:acpi:BOSC0200*:dmi:*svnAcer*:*pnSpinSP111-34:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
 
+#########################################
+# Aquarius
+#########################################
+
+# Aquarius NS483
+sensor:modalias:acpi:MXC6655*:dmi:*:svnAquarius*:pnNS483:*
+ ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
+
 #########################################
 # Archos
 #########################################
